test/libsolidity/semanticTests/exp_operator.sol

```diff
@@ -0,0 +1,9 @@
+contract test {
+    function f(uint a) public returns(uint d) { return 2 ** a; }
+    function g(uint a, uint b) public returns(uint c) { return a ** b; }
+}
+// ----
+// f(uint256): 1
+// -> 2
+// g(uint256, uint256): 1, 1
+// REVERT # Argument encoding is not properly implemented, yet.
```

Comment:
Will the comment stay after a call to "update"?

Reply:
It might be easier to add the comment to the function call and not the the expectation.

Reply:
Hm, in general, it might make sense to allow newlines (which will be preserved during update) in the list of arguments. Is that supported?

Reply:
Basically this would mean that function call is everything up to a line that starts with `REVERT` or `->`.

Reply:
New lines for function calls are not supported as of now. To clarify, your proposal is to have the call and the expectations written down like this:
```
// f(uint256)
// 1
// -> 2
// g(uint256, uint256)
// 1, 1
// REVERT # Argument encoding is not properly implemented, yet.
```
Is that correct? I'd also slightly tend towards that solution as it would have the benefit to support not only Ether that is send (currently: `f(uint256): 1 [etherValue]`), but also argument ranges that I'd like to implement at some point:
```
// f(uint256, uint256): [etherValue]
// [1...10], 1
// -> 2
```
In my opinion, this would be consistent as everything that is send with the function call (arguments and ether) is encoded in brackets. But maybe that's a bit confusing and we should keep the ether value in brackets only. Supporting argument ranges could simply work like this: `1..10, 1`.

Reply:
> It might be easier to add the comment to the function call and not the the expectation.
The comment is preserved on both lines already. So I guess it's fine for now. Would need to make sure that this works if we're going to add another extra line for the arguments.

Reply:
Ah, I was suggesting a relaxed handling of newline characters: I would assume that sometimes, the arguments can be rather large and the argument list long, so you would want to be able to insert newlines at convenient places.

Reply:
After reading the comment again, I also got your intention.